UKHomeOffice/nmsw-ui
# NMSW-1195 enable submit

# Ticket



----

## AC

Re-enable submit button

----

## To test

- create a declaration through to submission
- > submit button should exist
- click submit
- > declaration should submit

----

## Notes
